dotcom-rendering/src/web/experiments/lib/ab-exclusions.ts

```diff
@@ -47,3 +46,64 @@ export const setOrUseParticipations = (
 		return participations[abTestId]?.variant === variantId;
 	}
 };
+
+/**
+ * Regional Exclusions
+ */
+export const isInRegion = (region: string[]): boolean => {
+	const countryCode = getCountryCodeSync();
+	return !!countryCode && region.includes(countryCode);
+};
```

Comment:
Does this need to be synchronous?

What’s the benefit of this abstraction over an `includes`? 

```ts
const regions: CountryCode = ["US", "CA"];
regions.includes(getCountryCodeSync() ?? "");
```

Reply:
Fair point. I'm happy to close this PR and just use getCountryCodesSync in the test def